utils/utils.py

```diff
@@ -331,4 +319,31 @@ def read_csv(csv_file_name):
         list_values = sorted(list_values, key=lambda k: k['dataset'])
     except TypeError:
         list_values
-    return list_values
\ No newline at end of file
+    return list_values
+
+
+def add_metadata_from_raster_to_sample(sat_img_arr: np.ndarray,
```

Comment:
worried about the effect of processing raster with no metadata. This function has be used in a condition statement to account for raster without metadata or incorrect meta.

Reply:
see previous answer